airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -115,6 +119,25 @@ public TemporalClient(final WorkflowClient client, final Path workspaceRoot) {
             connectionId));
   }
 
+  public void submitConnectionUpdaterAsync(final long jobId, final int attemptId) {
+    final ConnectionUpdaterWorkflow connectionUpdaterWorkflow = getWorkflowStub(ConnectionUpdaterWorkflow.class,
+        TemporalJobType.CONNECTION_UPDATER);
+    final ExecutorService threadpool = Executors.newCachedThreadPool();
+    final Future<Void> futureTask = threadpool.submit(() -> {
+      final JobRunConfig jobRunConfig = TemporalUtils.createJobRunConfig(jobId, attemptId);
+      execute(jobRunConfig, () -> connectionUpdaterWorkflow.run());
+
+      return null;
+    });
+
+    try {
+      Thread.sleep(10000);
```

Comment:
Can we make this wait until the workflow is run instead of using a fixed timeout?

Reply:
Done, I am using signalWithStart.